tests/utils/subnet.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/onsi/gomega"
 )
 
-func RunHardhatTests(test string, rpcURI string) {
+func RunTestCMD(testCMD *exec.Cmd, rpcURI string) {
```

Comment:
Does this need to be external or do we only use `RunDefaultHardHatTests` ? If exported, could we add a comment?

Reply:
`RunTestCMD` could be used if paths and test commands are different. Developers can customize their commands to run different tests (not only hardhat) in different networks (not only in local). I added more comments.